.gitignore

```diff
@@ -26,6 +26,7 @@ yarn-error.log*
 
 # local env files
 .env*.local
+.env
```

Comment:
My guess here is you ran `git add` before adding this .. next time make sure to add this to `.gitignore` before running `git add`  ... you did the opposite.